JS/Lesson15/emrah/script.js

```diff
@@ -1,164 +1,201 @@
+const planPriceString = "Premium plan - 9.99 USD/month."; // Should return "9.99 USD"
 
-console.log('Exercise16: ' /* Result here */);
-console.log('------------------------');
+console.log("Exercise16: ", planPriceString.slice(12, 23));
+console.log("------------------------");
 
 // 17. Use the `repeat` method to create a string of asterisks (*) that forms a horizontal line with a length of 10 and print it to the console.
 
-console.log('Exercise17: ', /* Result here */);
+console.log("Exercise17: ", "*".repeat(10));
 
-console.log('------------------------');
+console.log("------------------------");
 // 18. Use the `concat` method to combine three separate strings into one and print the result to the console.
-const string1Exercise18 = 'Hello, ';
-const string2Exercise18 = 'world';
-const string3Exercise18 = '!';
-console.log('Exercise18: ', /* Result here */);
-
-console.log('------------------------');
-// 19. Use the `split` and `join` methods to reverse the words in the string stringExercise28 and print the result to the console.
-const stringExercise19 = 'This is a sample sentence.';
-console.log('Exercise19: ', /* Result here */);
-
-console.log('------------------------');
+const string1Exercise18 = "Hello, ";
+const string2Exercise18 = "world";
+const string3Exercise18 = "!";
+console.log(
+  "Exercise18: ",
+  string1Exercise18.concat(string2Exercise18, string3Exercise18)
+);
+
+console.log("------------------------");
+// 19. Use the `split` and `join` methods to reverse the words in the string stringExercise19 and print the result to the console.
+const stringExercise19 = "This is a sample sentence.";
+const words = stringExercise19.split(" ");
+const reversedString = words.reverse().join(" ");
+console.log("Exercise19: ", reversedString);
+
+console.log("------------------------");
 
 // 20. Use the `charAt` method to extract the first and last characters of a string and concatenate them into a new string. Then use the `toUpperCase` method to make the new string uppercase and print the result to the console.
-const stringExercise20 = 'JavaScript';
-
-console.log('Exercise20: ', /* Result here */);
-
-console.log('------------------------');
+const stringExercise20 = "JavaScript";
+const firstCharacter = stringExercise20.charAt(0);
+const lastCharacter = stringExercise20.charAt(stringExercise20.length - 1);
+const newString = firstCharacter + lastCharacter;
+const uppercaseString = newString.toUpperCase();
+console.log("Exercise20: ", uppercaseString);
+
+console.log("------------------------");
 // 21. Use the `substring` method to extract the substring "quick brown fox" from the string stringExercise32 based on two indices and print it to the console.
-const stringExercise21 = 'The quick brown fox jumps over the lazy dog.';
-
-console.log('Exercise21: ', extractedSubstringExercise32);
+const stringExercise21 = "The quick brown fox jumps over the lazy dog.";
+const startIndex = 4;
```

Comment:
A variable with the name 'startIndex' has already been declared in this file. You have to use a different name for this variable otherwise code will not work.

Reply:
Please use dynamic indexes here as well